local/filesystem_utils.py

```diff
@@ -14,3 +15,13 @@ def append_file_list_mode(file_list, append_modes):
 
 def make_file_executable(file):
     append_file_mode(file, stat.S_IEXEC)
+
+def read_files_from_path(path, file_match_regex='.*\.md$'):
```

Comment:
find and duplications in reading directories content and apply this function to them